tests/basic.test.js

```diff
@@ -5,6 +5,7 @@ const { waitForConnectionOpen, awaitNextMessage, awaitAuth, awaitNextCommand, aw
 const server = new WebSocketServer({ port : 8081 });
 
 beforeAll(() => server.init());
+beforeEach(async () => await server.resetDataSet());
```

Comment:
returned promise is awaited by the beforeEach
```suggestion
beforeEach(() => server.resetDataSet());
```